plugins/docker/_docker

```diff
@@ -3023,6 +3023,12 @@ _dockerd() {
 
 _docker "$@"
 
+# aliases 
+# remove docker intermediate images
+alias drmi='docker rmi -f  $(docker images -aq --filter dangling=true) 2> /dev/null'
```

Comment:
What is the benefit of this command over `docker image prune` out of interest? As far as I know only dangling images are removed (in version 19.03 at least)